src/Confluent.Kafka/Admin/ElectLeadersException.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
```

Comment:
Use two newlines before all namespaces, it's a rule that has been constantly enforced in this client, and I think it's in the editorconfig PR too.

Reply:
Add this using for the change below 
```C#
using System.Linq;
```